flink-dist/src/main/flink-bin/bin/flink-console.sh

```diff
@@ -116,4 +118,11 @@ echo $$ >> "$pid" 2>/dev/null
 # Evaluate user options for local variable expansion
 FLINK_ENV_JAVA_OPTS=$(eval echo ${FLINK_ENV_JAVA_OPTS})
 
+if [ "${STD_REDIRECT}" == "true" ]; then
+  # disable console appender to avoid redundant logs in log file and out file
```

Comment:
We never have redundant logs in log file. Right?
Here is only to avoid the redundant logs in out file.